tests/small/LandIce/SHMIP/input_suite_A6.yaml

```diff
@@ -57,7 +57,9 @@ ANONYMOUS:
         Lump Mass: false
         Use Melting: false
       Darcy Law:
-        Regularize With Continuation: true
+        Regularization:
```

Comment:
Can you add a Regression test?

Reply:
This test did not converge (and according to Matt, this test case kind of requires channels, so it's ok that it didn't). I will remove the input file, to avoid confusion.